### PR TITLE
Tasks 886-889: Quail, Rotten Tomatoes and Go Emotions Dataset

### PR DESCRIPTION
**Pull request by saianirud**

### CODE REVIEW

File: tasks/task886_quail_question_generation.json [hunk 1]
```diff
@@ -0,0 +1,3048 @@
...
+            "output": [
+                "Who asked about Tibet?"
```
Comment (danyaljj) [inline tasks/task886_quail_question_generation.json:65]: There is only a single question per passage? I was expecting multiple.

File: tasks/task886_quail_question_generation.json [hunk 2]
```diff
@@ -0,0 +1,3048 @@
...
+    "Contributors": [
+        "Saianirud Reddy Nellore"
+    ],
+    "Source": [
+        "Quail(Link - https://huggingface.co/datasets/quail)"
+    ],
+    "Categories": [
+        "Question Generation"
+    ],
+    "Definition": "In this task, you are given a passage and your task is to generate a question from the passage such that it identifies a Character such as a person or a thing in the passage.",
```
Comment (danyaljj) [inline tasks/task886_quail_question_generation.json]: Maybe a bit simpler (less verbose)?

```
Given a passage, generate a question such that it identifies a character (a person or an object) in the passage.
```

Comment (danyaljj) [reply tasks/task886_quail_question_generation.json]: Similarly, please simplify the definition of the rest of the tasks.

**Comment by aarunku5** [comment]
@saianirud can you update your tasks accordingly? Thanks!

**Comment by danyaljj** [comment]
Please make sure to set the `"Categories": [""]` and `"Domains": [""]` fields according to [the taxonomy presented here](https://github.com/allenai/natural-instructions-expansion/blob/master/doc/task-hierarchy.md).

**Comment by swarooprm** [comment]
@saianirud, a gentle reminder!

**Comment by swarooprm** [comment]
> @saianirud, a gentle reminder!

@saianirud, wondering if you plan to update?

**Comment by danyaljj** [comment]
- Applied some rephrasing to the task instructions. 
 - Moved one improvement to #489

**Pull request merged**